packages/shared/src/utils/helpers/duration.ts

```diff
@@ -67,7 +67,7 @@ export const buildDurationConfig = (
 
     const duration_maps = getDurationMaps();
 
-    if (/^tick|daily$/.test(contract.expiry_type)) {
+    if (/^(?:tick|daily)$/.test(contract.expiry_type)) {
```

Comment:
Fix for:
<img width="782" alt="Screenshot 2023-07-19 at 3 27 27 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/756810f1-5dc0-44c2-9213-aa6b023d070b">